parse.py

```diff
@@ -879,6 +892,10 @@ def _handle_field(self, field):
         if type in self._extra_types:
             type_converter = self._extra_types[type]
             s = getattr(type_converter, 'pattern', r'.+?')
+            # -- EXTENSION: group_count attribute
+            group_count = getattr(type_converter, 'group_count', 0)
```

Comment:
Please add some documentation about "group_count" and what it does to the README section about custom types. Also, I think "regex_group_count" would be a clearer name for the attribute.

Also for the user experience I feel like we should handle "regex_group_count" in the with_pattern decorator, rather than require manual setting of the attribute.

Reply:
Ok, I can do that.

REGARDING: README
Note that the README is not up-to-date w/ the parse module docstrings.
If you run "python setup.py sdist" the README file is regenerated from the parse module.
Note after this action, the README differs from the checked-in version even without my changes.